client/my-sites/subscribers/components/grow-your-audience/grow-your-audience.tsx

```diff
@@ -60,17 +104,10 @@ const GrowYourAudience = () => {
 						'Create fresh content, publish regularly, and understand your audience with site stats.'
 					) }
 					title={ translate( 'Keep your readers engaged' ) }
+					ctaLabel={ translate( 'Learn more' ) }
+					externalUrl
```

Comment:
For these externalUrls, should we consider linking to the support modal on this page, rather than redirecting away to the support site? The support site may be a bit easier for reading/digesting lots of information. But the modal popup is nice in that it keeps users here, in flow.

Reply:
Generally agreed, but this articles is at https://wordpress.com/go/ so less of a support article, and more of a lengthier tutorial. I think this one is better at new tab.